airbyte-integrations/connectors/source-mysql/metadata.yaml

```diff
@@ -1,7 +1,7 @@
 data:
   ab_internal:
     ql: 300
-    sl: 100
+    sl: 200
```

Comment:
oh, it shoud be 300 then. we are certifying as Bronze. Updated PR title to be clear.